### PR TITLE
Add a link to GET endpoints without arguments.

### PR DESCRIPTION
**Pull request by jwg4**
This means that its very easy to see the contents of an endpoint like this.

### CODE REVIEW

File: flask_autodoc/templates/autodoc_default.html [hunk 1]
```diff
@@ -57,9 +57,15 @@
...
             <ul class="methods">
-                    {% for method in doc.methods -%}
-                    <li class="method">{{method}}</li>
-                    {% endfor %}
+                {% for method in doc.methods -%}
+                    {% if method == 'GET' and doc.args == ['None'] %}
+                        <a href="{{doc.rule}}" class="getmethod">
+                    {% endif %}
+                        <li class="method">{{method}}</li>
+                    {% if method == 'GET' %}
```
Comment (lukeyeager) [inline flask_autodoc/templates/autodoc_default.html]: Shouldn't you copy the same `if` condition on this line? What happened to `doc.args`?

It seems like an `if/else` would be easier to read, even if it means duplicating the `<li>` section.

**Comment by jwg4** [comment]
You were right about the second condition which was an error. Have replaced with if + else. Happy to squash the commits if you prefer, haven't done so yet to keep the discussion clear in case there are more comments.

**Comment by lukeyeager** [comment]
Assigning myself to try this out when I find some time ...

**Comment by lukeyeager** [comment]
LGTM. Here's a screenshot of the new functionality (note the links):

autodoc-get-link-pr

Thanks @jwg4!

**Pull request merged**